src/common/styles/themes.tsx

```diff
@@ -0,0 +1,253 @@
+import React, { ReactElement } from "react";
+import { ViewStyle } from "react-native";
+import AppLogo from "../../../assets/Logo.svg";
+import GovWalletAppLogo from "../../../assets/gw-logo.svg";
+import { color } from "../../common/styles";
+
+export type CustomTheme = {
+  name: string;
+  logo: () => ReactElement;
+  appTextColor: string;
+  topBackground: {
+    production: {
+      primaryColor: string;
+      secondaryColor: string;
+    };
+    staging: {
+      primaryColor: string;
+      secondaryColor: string;
+    };
+  };
+  darkButton: {
+    enabled: {
+      borderColor: ViewStyle["borderColor"];
+      backgroundColor: ViewStyle["backgroundColor"];
+    };
+    disabled: {
+      borderColor: ViewStyle["borderColor"];
+      backgroundColor: ViewStyle["backgroundColor"];
+    };
+  };
+  secondaryButton: {
+    enabled: {
+      borderColor: ViewStyle["borderColor"];
+      textColor: string;
+    };
+    disabled: {
+      borderColor: ViewStyle["borderColor"];
+      textColor: string;
+    };
+  };
+  drawer: {
+    openButtonColor: string;
+    generalButtonColor: string;
+  };
+  checkoutSuccessCard: {
+    backgroundColor: string;
+    thumbsUpIconColor: string;
+  };
+  checkoutUnsuccessfulCard: {
+    backgroundColor: string;
+    thumbsDownIconColor: string;
+  };
+  customerCard: {
+    userIconColor: string;
+    successfulHeaderColor: ViewStyle["backgroundColor"];
+    unsuccessfulHeaderColor: ViewStyle["backgroundColor"];
+    idTextColor: string;
+    idLabelColor: string;
+  };
+  collectCustomerDetails: {
+    statsIconColor: string;
+    needHelpIcon: string;
+    needHelpText: string;
+  };
+  //TODO: check if we wanna change id input at all
+  idInput: {
+    inputMinWidth: string;
+    inputAndButtonFlexDirection: string;
+  };
+  statisticsScreen: {
+    topBackgroundColor?: string;
+    headerBackButtonColor: string;
+    headerTextColor: string;
+    headerDrawerButtonColor: string;
+    statTextColor: string;
+    smallTextColor: string;
+    dateTextColor: string;
+    enabledChevron: string;
+    enabledChevronOpacity: number;
+    disabledChevron: string;
+    disabledChevronOpacity: number;
+  };
+  alertModal: {
+    titleColor: string;
+  };
+};
+
+export const govWalletTheme: CustomTheme = {
+  name: "GOVWALLET",
+  // eslint-disable-next-line react/display-name
```

Comment:
Why is this necessary?

Reply:
the linter is complaining since the function returning the logo component isn't named. Wasn't sure if we needed to name this function tho since it's returning a static image! Wdyt?

Reply:
I think it's fine!